python/lsst/obs/base/_fitsRawFormatterBase.py

```diff
@@ -368,7 +368,7 @@ def readFull(self, parameters=None):
     def readRawHeaderWcs(self, parameters=None):
         """Read the SkyWcs stored in the un-modified raw FITS WCS header keys.
         """
-        return lsst.afw.geom.makeSkyWcs(lsst.afw.fits.readMetadata(self.fileDescriptor))
+        return lsst.afw.geom.makeSkyWcs(lsst.afw.fits.readMetadata(self.fileDescriptor.location.path))
```

Comment:
Did this ever work?  I'm not asking for more unittests, but how do you know that it's correct now?

Reply:
I'm pretty sure it's never worked and can't have ever worked because self.fileDescriptor is not a path to a file. The form with location.path is the form used in the rest of the code. It does mean that this code is not tested anywhere and it's not called from anything that I can find. I think this method is more aspirational for the future -- gen3 currently can't do the header_wcs derived component that gen2 can do and we have to add it at some point.